client/additional-methods-setup/methods-selector/add-payment-methods-task.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback, useContext, useState } from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import WizardTaskContext from '../wizard/task/context';
+import CollapsibleBody from '../wizard/collapsible-body';
+import WizardTaskItem from '../wizard/task-item';
+import PaymentMethodCheckboxes from '../../components/payment-methods-checkboxes';
+import PaymentMethodCheckbox from '../../components/payment-methods-checkboxes/payment-method-checkbox';
+import './add-payment-methods-task.scss';
+
+const useGetCountryName = () => {
+	const baseLocation = useSelect(
+		( select ) =>
+			select( 'wc/admin/settings' ).getSetting(
```

Comment:
Should we use [@woocommerce/data](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/data/README.md) ?
`import { SETTINGS_STORE_NAME } from '@woocommerce/data';`
...
`select( SETTINGS_STORE_NAME ).getSetting( 'wc_admin'`, ...

Reply:
Thanks for the explanation.
Looks like the module `md5` is included with `@woocommerce/components` but not with the version we are using.
For the future then.